lib/LedgerSMB/Template/UI.pm

```diff
@@ -32,7 +32,7 @@ our $singleton;
 our @pre_render_cbs = (
     sub {
         my ($request, $template, $vars, $cvars) = @_;
-        $vars->{USER} = $request->{_user};
+        $vars->{USER} = $request->{_user} // { language => LedgerSMB::Sysconfig::language() };
```

Comment:
Why do we need this one? You handled setup and login in the other two changes...

Reply:
_user is only set for a logged in user

Reply:
True, but in which situations does that happen, now that you override the content of `_user` in login and setup handling?

Reply:
Every time a user is not logged. I only override `{user}` default language in login and setup, not `{_user}`.

Reply:
Are you telling me we have both `user` *and* `_user` keys in `$request`? can we eliminate `user` in favor of having only `_user`?

Reply:
Fixed